main.go

```diff
@@ -99,9 +99,13 @@ func main() {
 					})
 				case *ast.ValueSpec:
 					for _, id := range spec.Names {
+						varOrConst := "variable"
+						if "const" == decl.Tok.String() {
```

Comment:
Is `decl.Tok` guaranteed to contain a value always? Should we guard against the case where the `Tok` is null?

Reply:
by looking into https://github.com/golang/go/blob/master/src/go/token/token.go not seeing any null/nil case, the token returned will be numerical string on last resort, should be safe imo. 
```func (tok Token) String() string {
	s := ""
	if 0 <= tok && tok < Token(len(tokens)) {
		s = tokens[tok]
	}
	if s == "" {
		s = "token(" + strconv.Itoa(int(tok)) + ")"
	}
	return s
}
```

Reply:
@stamblerre thoughts?

Reply:
`decl.Tok` will always have a value (see https://pkg.go.dev/go/ast?tab=doc#GenDecl.Tok), and it cannot be nil, since a `token.Token` is an int. However, the correct way to check its value would be: `if decl.Tok == token.VAR` and similarly for the `token.CONST` case.

Reply:
Thanks @stamblerre 

@piengeng Can you make the suggested change?

Reply:
Thanks @stamblerre!

@piengeng Can you make the suggested change?